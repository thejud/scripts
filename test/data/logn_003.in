1
2.718281828459045
10
