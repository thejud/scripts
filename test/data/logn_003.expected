0.0
1.0
2.302585092994046
